src/workloads/scale/NegativeScalingLoadStress.yml

```diff
@@ -0,0 +1,126 @@
+  has been shown to identify negative scaling on dual socket instances and was originally created to
+  mimic the behvaiour reported in HELP-44821. This workload is not scheduled to run at this time and is
+  intended for adhoc use.
+
+# This workload does not support sharding yet.
+
+Keywords:
+- scale
+- insertOne
+- insert
+- findOne
+- find
+
+dbname: &dbname mix
+CollectionCount: &NumColls 512
+PhaseDuration: &PhaseDuration 1 minute
+Document: &doc
+  _id: {^Inc : {}}
+  sortKey: {^Inc : {}}
+  accountId: {^FastRandomString: {length: 10}}
+  accountOfficerId: 1
+  amountInAccountCurrency: {^RandomDouble: { min: 0.0, max: 1000.0 }}
+  amountInEventCurrency: {^RandomDouble: { min: 0.0, max: 1000.0 }}
+  bookingDate: {^RandomDate: {min: "2020-01-01", max: "2023-01-01"}}
+  categorisactionId: 130
+  chequeNumber: 123
+  currency: "USD"
+  customerId: { ^RandomInt: { min: 10000, max: 100000 } }
+  customerReference: "FT21105YDXJT"
+  extensionData: {}
+  externalIndicator: false
+  externalReference: "XYZ"
+  narrative: "ABC"
+  objectID: {^FastRandomString: {length: 25}}
+  processingDate: {^RandomDate: {min: "2020-01-01", max: "2023-01-01"}}
+  recordId: 197453644225786.030004
+  runningBalance: {^RandomDouble: { min: 0.0, max: 1000.0 }}
+  transactionAmount: {^RandomDouble: { min: 0.0, max: 1000.0 }}
+  transactionReference: "FT21105YDXJT"
+  valueDate: {^RandomDate: {min: "2020-01-01", max: "2023-01-01"}}
+
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2500
+  Insert:
+    QueryOptions:
+      maxPoolSize: 2500
+
+ActorTemplates:
+
+- TemplateName: InsertTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "Insert"}}
+    Type: CrudActor
+    Database: *dbname
+    ClientName: Insert
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 1}}]
+        NopInPhasesUpTo: 2
+        PhaseConfig:
+          Repeat: 100000
+          RecordFailure: true
+          CollectionCount: *NumColls
+          Operations:
+          - OperationName: insertOne
+            OperationCommand:
+              Document: *doc
+
+- TemplateName: FindTemplate
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "Find"}}
+    Type: CrudActor
+    Database: *dbname
+    ClientName: Default
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 2}}]
+        NopInPhasesUpTo: 2
+        PhaseConfig:
+          Repeat: 100000
+          RecordFailure: true
+          CollectionCount: *NumColls
+          Operations:
+          - OperationName: findOne
+            OperationCommand:
+              Filter: {_id: {^RandomInt: {min: 0, max: 100000}}}
+
+Actors:
+
+# Insert Actors
+- ActorFromTemplate:
+    TemplateName: InsertTemplate
+    TemplateParameters:
+      Name: Insert_1024
```

Comment:
Should this _1024 within the name be updated? 
I think it is usually either the number of docs or threads